src/osara.h

```diff
@@ -207,6 +207,7 @@
 #define REAPERAPI_WANT_FreeHeapPtr
 #define REAPERAPI_WANT_TimeMap2_timeToQN
 #define REAPERAPI_WANT_MIDI_GetGrid
+#define REAPERAPI_WANT_GetParentTrack
```

Comment:
This is requested but not used. However, I wonder whether we can use it after all; see below.